examples/predictors/linear/Example-Train-Linear-Rollout-Model.ipynb

```diff
@@ -478,7 +478,21 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### 2 months"
+    "## 180 days, from a future date, all countries and regions\n",
```

Comment:
This one is the most important test case: it checks the predictor can predict in the future, for 180 days, and for all countries

Reply:
I noticed `lstm` doesn't predict for all countries -- what causes it not to generate predictions for some? (I noticed it because RoboJudge outputs debug info when it's filling in missing predictions)

Reply:
That's because it has not population data for som in `Additional_Context_Data_Global.csv`
Like for `Nepal`, `Somalia`, etc.
I'll see what we can do about it in #92

Reply:
For Taiwan it turns out the Oxford dataset contains only NaN for ConfirmedCases and ConfirmedDeaths